zio-schema/shared/src/main/scala/zio/schema/StandardType.scala

```diff
@@ -1,43 +1,130 @@
 package zio.schema
 
+import zio.Chunk
+
 import java.time._
 import java.time.format.DateTimeFormatter
 import java.time.temporal.TemporalUnit
 
-import zio.Chunk
-
-sealed trait StandardType[A]
+sealed trait StandardType[A] {
+  def defaultValue: Either[String, A]
+}
 
 object StandardType {
-  implicit object UnitType   extends StandardType[Unit]
-  implicit object StringType extends StandardType[String]
-  implicit object BoolType   extends StandardType[Boolean]
-  implicit object ShortType  extends StandardType[Short]
-  implicit object IntType    extends StandardType[Int]
-  implicit object LongType   extends StandardType[Long]
-  implicit object FloatType  extends StandardType[Float]
-  implicit object DoubleType extends StandardType[Double]
-  implicit object BinaryType extends StandardType[Chunk[Byte]]
-  implicit object CharType   extends StandardType[Char]
-
-  implicit object BigDecimalType extends StandardType[java.math.BigDecimal]
-  implicit object BigIntegerType extends StandardType[java.math.BigInteger]
+  implicit object UnitType extends StandardType[Unit] {
+    def defaultValue: Either[String, Unit] = Right(())
+  }
+
+  implicit object StringType extends StandardType[String] {
+    def defaultValue: Either[String, String] = Right("")
+  }
+
+  implicit object BoolType extends StandardType[Boolean] {
+    def defaultValue: Either[String, Boolean] = Right(true)
```

Comment:
I think Bool should default to `false` to match the behavior of an uninitialized `Boolean` in Java and Scala.

Reply:
👍